backend/prompt_studio/prompt_studio_registry_v2/prompt_studio_registry_helper.py

```diff
@@ -50,28 +50,28 @@ def frame_spec(tool: CustomTool) -> Spec:
         properties = {
             "challenge_llm": {
                 "type": "string",
-                "title": "Challenge LLM",
+                "title": "Challenger LLM",
                 "adapterType": "LLM",
-                "description": "LLM to use for challenge",
+                "description": "LLM to use for LLMChallenge",
                 "adapterIdKey": "challenge_llm_adapter_id",
             },
             "enable_challenge": {
                 "type": "boolean",
-                "title": "Enable challenge",
+                "title": "Enable LLMChallenge",
                 "default": False,
-                "description": "Enables Challenge",
+                "description": "Enables LLMChallenge",
             },
             "summarize_as_source": {
                 "type": "boolean",
-                "title": "Summarize and use summary as source",
+                "title": "Enable SummarizedExtraction",
                 "default": False,
-                "description": "Enables summary and use summarized content as source",
+                "description": "Enables SummarizedExtraction",
             },
             "single_pass_extraction_mode": {
                 "type": "boolean",
-                "title": "Enable Single pass extraction",
+                "title": "Enable SinglePass Extraction",
```

Comment:
@jagadeeswaran-zipstack For summarization, we've named it "SummarizedExtraction" (with no space), while for SinglePass, we've named it "SinglePass Extraction" (with a space). Is this expected? Shouldn't we maintain consistency in the naming?

Reply:
@shuveb would be right person to answer here.

Reply:
Yes @tahierhussain. That's what I've used consistently everywhere in the documentation.